GiTinder/Data/GiTinderContext.cs

```diff
@@ -0,0 +1,51 @@
+﻿using GiTinder.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+
+namespace GiTinder.Data
+{
+    public class GiTinderContext : DbContext
```

Comment:
@FilipTeply ah, so you have another one, remove the first one then

Reply:
Done.